python/mxnet/gluon/nn/activations.py

```diff
@@ -0,0 +1,214 @@
+
+# coding: utf-8
+# pylint: disable= arguments-differ
+"""Basic neural network layers."""
+__all__ = ['Activation', 'LeakyReLU', 'PReLU', 'ELU', 'SELU', 'Swish']
+
+from ..block import HybridBlock
+
+
+class Activation(HybridBlock):
+    r"""Applies an activation function to input.
+
+    Parameters
+    ----------
+    activation : str
+        Name of activation function to use.
+        See :func:`~mxnet.ndarray.Activation` for available choices.
+
+
+    Inputs:
+        - **data**: input tensor with arbitrary shape.
+
+    Outputs:
+        - **out**: output tensor with the same shape as `data`.
+    """
+    def __init__(self, activation, **kwargs):
+        self._act_type = activation
+        super(Activation, self).__init__(**kwargs)
+
+    def _alias(self):
+        return self._act_type
+
+    def hybrid_forward(self, F, x):
+        return F.Activation(x, act_type=self._act_type, name='fwd')
+
+    def __repr__(self):
+        s = '{name}({_act_type})'
+        return s.format(name=self.__class__.__name__,
+                        **self.__dict__)
+
+
+class LeakyReLU(HybridBlock):
+    r"""Leaky version of a Rectified Linear Unit.
+
+    It allows a small gradient when the unit is not active
+
+    .. math::
+
+        f\left(x\right) = \left\{
+            \begin{array}{lr}
+               \alpha x & : x \lt 0 \\
+                      x & : x \geq 0 \\
+            \end{array}
+        \right.\\
+
+    Parameters
+    ----------
+    alpha : float
+        slope coefficient for the negative half axis. Must be >= 0.
+
+
+    Inputs:
+        - **data**: input tensor with arbitrary shape.
+
+    Outputs:
+        - **out**: output tensor with the same shape as `data`.
+    """
+    def __init__(self, alpha, **kwargs):
+        assert alpha >= 0, "Slope coefficient for LeakyReLU must be no less than 0."
+        super(LeakyReLU, self).__init__(**kwargs)
+        self._alpha = alpha
+
+    def hybrid_forward(self, F, x):
+        return F.LeakyReLU(x, act_type='leaky', slope=self._alpha, name='fwd')
+
+    def __repr__(self):
+        s = '{name}({alpha})'
+        return s.format(name=self.__class__.__name__,
+                        alpha=self._alpha)
+
+
+class PReLU(HybridBlock):
+    r"""Parametric leaky version of a Rectified Linear Unit.
+    <https://arxiv.org/abs/1502.01852>`_ paper.
+
+    It learns a gradient when the unit is not active
+
+    .. math::
+
+        f\left(x\right) = \left\{
+            \begin{array}{lr}
+               \alpha x & : x \lt 0 \\
+                      x & : x \geq 0 \\
+            \end{array}
+        \right.\\
+
+    where alpha is a learned parameter.
+
+    Parameters
+    ----------
+    alpha_initializer : Initializer
+        Initializer for the `embeddings` matrix.
+
+
+    Inputs:
+        - **data**: input tensor with arbitrary shape.
+
+    Outputs:
+        - **out**: output tensor with the same shape as `data`.
+    """
+    def __init__(self, alpha_initializer='zeros', *args):
```

Comment:
Is 0 initialization standard?

Reply:
[Tensorflow/Keras uses zeros](https://www.tensorflow.org/api_docs/python/tf/keras/layers/PReLU#__init__), [Pytorch uses 0.25](http://pytorch.org/docs/master/nn.html?highlight=prelu#torch.nn.PReLU)

Do we have a constant initializer to achieve the latter?